src/cli.ts

```diff
@@ -118,6 +118,10 @@ if (require.main === module) {
       '-e, --electron-version <value>',
       "electron version to package, without the 'v', see https://github.com/electron/electron/releases",
     )
+    .option(
+      '--wvvmp',
+      'use widevine enabled beta version of electron',
```

Comment:
```suggestion
      'use a Widevine-enabled version of Electron (use at your own risk, it's unofficial, provided by CastLabs)',
```

Reply:
Need double quotes for this one...

```suggestion
      "use a Widevine-enabled version of Electron (use at your own risk, it's unofficial, provided by CastLabs)",
```

Reply:
👍